google/cloud/bigquery/client.py

```diff
@@ -2686,16 +2678,6 @@ def load_table_from_dataframe(
 
         try:
             if new_job_config.source_format == job.SourceFormat.PARQUET:
-                if _PYARROW_VERSION in _PYARROW_BAD_VERSIONS:
```

Comment:
Let's do a `try_import` with raise to catch out-of-date pyarrow here.

Reply:
I'm not sure about this one. At line #119, we have already done `try_import`, and this function only allows pyarrow > 3.0.0.

Reply:
done